js/src/modals/Verification/GatherData/gatherData.js

```diff
@@ -175,41 +179,41 @@ export default class GatherData extends Component {
     return (
       <p className={ styles.message }>
         <FormattedMessage
-          id='ui.verification.gatherData.isVerified.pending'
+          id='ui.verification.gatherData.accountIsVerified.pending'
           defaultMessage='Checking if your account is verified…'
         />
       </p>
     );
   }
 
   renderRequested () {
-    const { isVerified, hasRequested } = this.props;
+    const { accountIsVerified, accountHasRequested } = this.props;
 
     // If the account is verified, don't show that it has requested verification.
-    if (isVerified) {
+    if (accountIsVerified) {
       return null;
     }
 
-    if (hasRequested) {
+    if (accountHasRequested) {
       return (
         <div className={ styles.container }>
           <InfoIcon />
           <p className={ styles.message }>
             <FormattedMessage
-              id='ui.verification.gatherData.hasRequested.true'
-              defaultMessage='You already requested verification.'
+              id='ui.verification.gatherData.accountHasRequested.true'
+              defaultMessage='You already requested verification from this account.'
             />
           </p>
         </div>
       );
-    } else if (hasRequested === false) {
+    } else if (accountHasRequested === false) {
```

Comment:
it's a nullable bool. i know this is ugly, but spamming the `props` didn't feel right either. not happy with this 😕